tests/queries/0_stateless/02868_distinct_to_count_optimization.reference

```diff
@@ -0,0 +1,287 @@
+-- { echoOn }
+
+-- test simple distinct
+SELECT uniq(a) FROM (SELECT DISTINCT a FROM test_rewrite_uniq_to_count) settings allow_experimental_analyzer=0;
```

Comment:
Do we really need to put all the `Select ...` things (query text) in the reference?